server/model-config.json

```diff
@@ -26,6 +26,10 @@
     "dataSource": "db",
     "public": false
   },
+  "course": {
+    "dataSource": "db",
+    "public": true
```

Comment:
From @treyhunner according to http://docs.strongloop.com/display/public/LB/model-config.json , should be set to true so it's available for REST